python/mlx/nn/layers/normalization.py

```diff
@@ -4,6 +4,80 @@
 from mlx.nn.layers.base import Module
 
 
+class InstanceNorm(Module):
+    r"""Applies instance normalization [1] on the inputs.
+
+    Computes
+
+    .. math::
+
+        y = \frac{x - \mathrm{E}[x]}{ \sqrt{\mathrm{Var}[x] + \epsilon}} * \gamma + \beta,
+
+    where :math:`\gamma` and :math:`\beta` are learned per feature dimension
+    parameters initialized at 1 and 0 respectively. Both are of size num_features,
+    if :attr:`affine` is ``True``.
+
+    [1]: https://arxiv.org/abs/1607.08022
+
+    Args:
+        num_features: number of features of the input
+        eps: a value added to the denominator for numerical stability. Default: 1e-5
+        momentum: the value used for the running_mean and running_var computation. Default: 0.1
+        affine: a boolean value that when set to ``True``, this module has
+            learnable affine parameters, initialized the same way as done for batch normalization.
+            Default: ``False``.
```

Comment:
Minor nits: 
- Add types next to the arg names e.g. `dims (int):`
- upper case sentence start after the arg name e.g. `dims: The number of features of the input.`
- se period at end consistently. 
- Add  double backticks for the default argument values.